src/test/java/org/folio/processing/events/utils/ZIPArchiverTest.java

```diff
@@ -0,0 +1,29 @@
+
+  @Test
+  public void shouldArchiveAndUnarchiveString() throws IOException {
```

Comment:
zip() and unzip() are two different methods - it's better to have separate tests for them

Reply:
Sure. I added new tests.
But I think it will be better don`t remove this test-case, because it checks the full flow of this mechanism.